comps/cores/telemetry/opea_telemetry.py

```diff
@@ -13,6 +14,8 @@
 from opentelemetry.sdk.trace.export import BatchSpanProcessor
 from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
 
+ENABLE_OPEA_TELEMETRY = os.getenv("ENABLE_OPEA_TELEMETRY", "false").lower() == "true"
```

Comment:
Instead of using 2 env variable `ENABLE_OPEA_TELEMETRY`, `TELEMETRY_ENDPOINT` to control the tracing, can we use just single env var `TELEMETRY_ENDPOINT` to control the tracing. When `TELEMETRY_ENDPOINT` is empty/None, it means the tracing is OFF. And we should make the tracing by default OFF.

Reply:
I prefer not to because for one thing, the example side already use that ENABLE_OPEA_TELEMETRY flag
 to control whether to use telemetry, and I'm not sure whether removing it will cause some complaints from users that already use the telemetry. For another, I think a bit redundancy can help future contributors be easily aware of one fact that telemetry can be on/off with this flag. Otherwise we have to explain this rule in README or somewhere.